index.bs

```diff
@@ -292,8 +338,8 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
 
```

Comment:
We might want to revisit whether this step (above) is still necessary, and if so, point out that the behaviour diverges from CSP here.

Reply:
Added a note